### PR TITLE
difference from empty use [] or length=0

### PR DESCRIPTION
**Pull request by microlv**
FYI

[Review on Reviewable](https://reviewable.io/reviews/loverajoel/jstips/119)

### CODE REVIEW

**Comment by microlv** [comment]
Make Array empty/clear more performance

**Comment by homer0** [comment]
Nice and a simple :). I like it :skull:

File: README.md [hunk 1]
```diff
@@ -18,6 +18,50 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter.
...
+
+Think about what will output:
+```js
+var foo = [1,2,3];
+var bar = [1,2,3];
+var foo2 = foo;
+var bar2 = bar;
+foo = [];
+bar.length = 0;
+console.log(foo, bar, foo2, bar2);
```
Comment (loverajoel) [inline README.md:64]: hey can you add the console.log output here?

**Comment by loverajoel** [comment]
nice tip!

File: README.md (hunk 1 above)
Comment (microlv) [inline README.md:64]: sure.

**Comment by microlv** [comment]
@mallowigi Thanks for correct my language error. ;)

**Comment by loverajoel** [comment]
@microlv thanks for your tip!

**Comment by mrvicadai** [comment]
@loverajoel the title `#(number)` is still there (should be #22)

**Comment by loverajoel** [comment]
@mrvicadai I changed it in master, don't worry

**Pull request merged**